tox.ini

```diff
@@ -6,11 +6,11 @@ exclude = migrations/*,docs/*
 
 [tox]
 envlist =
-    py27-{1.8,1.9,master},
-    py33-{1.8},
-    py34-{1.8,1.9,master},
-    py35-{1.8,1.9,master},
-
+;    py27-{1.8,1.9,master},
+;    py33-{1.8},
+;    py34-{1.8,1.9,master},
+;    py35-{1.8,1.9,master},
```

Comment:
@arthur-wsw why did you commend out other tests matrix? This should go back I think.